core/scm.js

```diff
@@ -192,6 +192,10 @@ const SCHEMA_HOOK = Joi.object().keys({
 
     username: Joi.reach(SCHEMA_USER, 'username'),
 
+    commitAuthors: Joi.array()
```

Comment:
I think it'd be safer to specify what kind of things can be in the array, like https://github.com/screwdriver-cd/data-schema/blob/master/models/template.js#L15

Reply:
Thanks for info, I have changed it